chat/src/main/java/net/md_5/bungee/chat/BaseComponentSerializer.java

```diff
@@ -12,16 +12,16 @@
 import java.util.Collections;
 import java.util.IdentityHashMap;
 import java.util.Locale;
-import net.md_5.bungee.api.ChatColor;
 import net.md_5.bungee.api.chat.BaseComponent;
 import net.md_5.bungee.api.chat.ClickEvent;
+import net.md_5.bungee.api.chat.ComponentStyle;
 import net.md_5.bungee.api.chat.HoverEvent;
 import net.md_5.bungee.api.chat.hover.content.Content;
 
 public class BaseComponentSerializer
 {
 
-    private static boolean getAsBoolean(JsonElement el)
```

Comment:
I think this is only used for style stuff now?

Reply:
Yep, you're right. Thought there was another instance of it.